tests/runtests.py

```diff
@@ -50,10 +50,6 @@
 # patterns.
 warnings.simplefilter("error", ResourceWarning)
 warnings.simplefilter("error", RuntimeWarning)
-# Ignore known warnings in test dependencies.
-warnings.filterwarnings(
-    "ignore", "'U' mode is deprecated", DeprecationWarning, module="docutils.io"
-)
```

Comment:
We should bump the minimum supported version of `docutils` before removing this, check out `tests/requirements/py3.txt` and `docs/internals/contributing/writing-code/unit-tests.txt`.